docs/fuse-ufs.8

```diff
@@ -0,0 +1,88 @@
+.\" Copyright (c) 2024 Benjamin Stürz <benni@stuerz.xyz>
+.Dd $Mdocdate$
```

Comment:
What expands the Mdocdate macro?  It looks like a CVS/SVN tag.  But since you aren't using CVS or SVN, you should just manage the date manually.

Reply:
It expands to the file's timestamp (mtime?) AFAIK.

Reply:
Correction: it uses OpenBSD's cvs(1) to find out the version and it's therefore not portable.